WordPress/Classes/ViewRelated/Stats/Extensions/WPStyleGuide+Stats.swift

```diff
@@ -32,6 +50,24 @@ extension WPStyleGuide {
             label.font = summaryFont
         }
 
+        static func configureLabelAsSubtitle(_ label: UILabel) {
+            label.textColor = secondaryTextColor
+            label.font = subTitleFont
+            label.text = label.text?.localizedUppercase
+        }
+
+        static func configureLabelItemDetail(_ label: UILabel) {
```

Comment:
Do you expect those to grow in what configuration they do beyond just setting the text color? 

I'm not convinced how much useful information those one-liners convey compared to just setting the text color in `awakeFromNib` or something.

Reply:
> I'm not convinced how much useful information those one-liners convey compared to just setting the text color in awakeFromNib or something.

I see what you're saying. But my goal was actually to be able to configure an element without knowing the specifics of what it needs to be. For example, like here, if I wanted to configure a label as item detail, well then I just call that method instead of having to look up what the color should be.


> Do you expect those to grow in what configuration they do beyond just setting the text color?

Hrm, at this point, no. But we have _one_ place to change it to if it does.